### PR TITLE
Minor test enhancements

### PR DESCRIPTION
**Pull request by okurz**
* t: Turn comments of 06-operator_links into explicit test notes
* Delete obsolete reference to phantomjs
* t: Delete obsolete FIXME

### CODE REVIEW

File: assets/javascripts/shapes.js [hunk 1]
```diff
@@ -420,7 +420,7 @@ CanvasState.prototype.draw = function() {
       var lineWidth = 1;
       ctx.strokeStyle = 'white';
       ctx.lineWidth = lineWidth;
-      // plainly ignore in phantomjs
+      // plainly ignore in phantomjs/Selenium
```
Comment (Martchus) [inline assets/javascripts/shapes.js:423]: You can likely remove phantomjs completely.

File: t/ui/06-operator_links.t [hunk 2]
```diff
@@ -35,59 +35,48 @@ unless ($driver) {
...
 $test_case->login($t, 'percival');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = true;/, "test list rendered with is_operator");
-
-# List with a not authorized user
+note 'List with a not authorized user';
 $test_case->login($t, 'lancelot', email => 'lancelot@example.com');
 $t->get_ok('/tests')->status_is(200)->content_like(qr/is_operator = false;/, "test list rendered without is_operator");
-
-# now the same for scheduled jobs
+note 'now the same for scheduled jobs';
```
Comment (Martchus) [inline t/ui/06-operator_links.t:46]: This note is terribly written and after it we just logout before the next note is printed so it doesn't make much sense.

File: t/ui/06-operator_links.t [hunk 3]
```diff
@@ -35,59 +35,48 @@ unless ($driver) {
...
-# operator has access to part of admin menu - using phantomjs
+note 'operator has access to part of admin menu';
 $driver->title_is("openQA", "on main page");
 $driver->find_element_by_link_text('Login')->click();
-# we're back on the main page
+note 'we are back on the main page';
 $driver->title_is("openQA", "back on main page");
-# but ...
-
+note 'but ...';
```
Comment (Martchus) [inline t/ui/06-operator_links.t:61]: If you're changing things here please improve the phrasing.

File: t/ui/06-operator_links.t [hunk 4]
```diff
@@ -35,59 +35,48 @@ unless ($driver) {
...
 $driver->find_element_by_xpath('//tr[./td[@class="nick" and text()="Demo"]]/td[@class="role"]//label[2]')->click;
-
-# refresh and return to admin pages
+note 'refresh and return to admin pages';
 $driver->refresh;
 $driver->get($driver->get_current_url =~ s/users//r);
 
 $driver->find_element('#user-action a')->click();
-# we should see test templates, groups, machines
+note 'we should see test templates, groups, machines';
```
Comment (Martchus) [inline t/ui/06-operator_links.t:75]: Maybe some of these notes should be subtests instead, e.g. `subtest 'test templates, groups and machines shown' => `.

**Comment by okurz** [comment]
@Martchus will do the changes you proposed but I'd prefer to do it separately, first change existing comments in one commit, then change the phrasing

File: t/ui/06-operator_links.t (hunk 4 above)
Comment (okurz) [reply t/ui/06-operator_links.t:70]: I prefer to not introduce subtests here as the test module basically still follows one story from top to bottom, not multiple ones.

File: t/ui/06-operator_links.t (hunk 3 above)
Comment (okurz) [reply t/ui/06-operator_links.t]: done

File: t/ui/06-operator_links.t (hunk 2 above)
Comment (okurz) [reply t/ui/06-operator_links.t]: done

**Pull request merged**